pkgs/tools/networking/openconnect/default.nix

```diff
@@ -27,8 +28,9 @@ in stdenv.mkDerivation rec {
     "--without-openssl-version-check"
   ];
 
-  buildInputs = [ openssl gnutls gmp libxml2 stoken zlib ]
-    ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.PCSC;
+  buildInputs = [ openssl gnutls gmp libxml2 p11-kit stoken zlib ]
+    ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.PCSC
```

Comment:
darwin shouldn't be used as an input.

Reply:
you're saying that we should remove the darwin dependency ?
Note that I don't have the possibility to test under darwin...

Reply:
I mean like this https://github.com/NixOS/nixpkgs/blob/7248f89da2dab8bca3a9f4f11d0407d0c8e25e89/pkgs/top-level/all-packages.nix#L2899